<launch>
  <node name="orb_slam2_rgbd" pkg="orb_slam2_ros"
      type="orb_slam2_ros_rgbd" output="screen">

       <remap from="/camera/rgb/image_raw" to="/torso_front_camera/color/image_raw" />
       <remap from="/camera/depth_registered/image_raw" to="/torso_front_camera/aligned_depth_to_color/image_raw" />

       <param name="publish_pointcloud" type="bool" value="true" />
       <param name="publish_pose" type="bool" value="true" />
       <param name="localize_only" type="bool" value="true" />
       <param name="reset_map" type="bool" value="false" />

       <!-- static parameters -->
       <param name="load_map" type="bool" value="true" />
       <param name="map_file" type="string" value="pal.bin" />
       <param name="settings_file" type="string" value="$(find orb_slam2_ros)/orb_slam2/config/ARI2.yaml" />
       <param name="voc_file" type="string" value="$(find orb_slam2_ros)/orb_slam2/Vocabulary/ORBvoc.txt" />

       <param name="pointcloud_frame_id" type="string" value="orb_map" />
       <param name="camera_frame_id" type="string" value="tracked_camera_link" />
       <param name="min_num_kf_in_map" type="int" value="5" />
       
       <param name="correct_global_frame" type="bool" value="true" />
  </node>
</launch>
